src/components/Header.js

```diff
@@ -1,15 +1,62 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
 
+
+  
+let lang = 'en';
+
+function setLang(langCode) {
+  lang = langCode.target.id;
+  console.log(langCode.target.id)
+  document.getElementById('en').style.opacity = "0.5";
+  document.getElementById('es').style.opacity = "0.5";
+  document.getElementById('fr').style.opacity = "0.5";
+  document.getElementById(langCode.target.id).style.opacity = "1";
+}
+
 function Header() {
   return (
       <header className="header">
-        <Link className="header__logo-link" to="/">
-          <h1 className="header__title">Dofus</h1>
-          <h1 className="header__title header__title--accent">Craftlist</h1>
-        </Link>
+        <div >
+          <Link className="header__logo-link" to="/">
+            <h1 className="header__title">Dofus</h1>
+            <h1 className="header__title header__title--accent">Craftlist</h1>
+          </Link>
+        </div>
+        <div style={{marginTop: '2px'}}>
+          <div id="en" style={{display: 'inline-block', opacity: 1}} onClick={setLang}>
```

Comment:
Regarding the flags, the approach is nice, but I'd prefer something more discreet hahaha. Something like this, maybe? With a select.
![Group 1](https://user-images.githubusercontent.com/61996301/180644132-78d88b0a-bf0d-4c59-bce1-4043631ea193.png)